.gitignore

```diff
@@ -2,3 +2,5 @@
 /target
 
 *.iml
+
+/*.sc
```

Comment:
Not a fan of ignoring `*.sc` in this case. It is used in workshop 1, 1a and 2 quite frequently. We either rewrite them to *.md files (e.g. make another PR for this in the (near) future) or we remove the *.sc here.